### ISSUE TITLE
Could not connect to the Steam Network after using UWPHook - only offline mode HELP

### ISSUE THREAD

**Issue by emrk30** [BrianLima/UWPHook#89]
Hi,

So, all I wanted was to add my XGP game to steam (FH5). At first I used the new custom installation mode in game pass (that let you access game files), thinking that I could now simply add the exe file of the game to steam, but access refused, wtf??. Anyway, I did a little googling and found this app. Seemed simple enough I decided to use it, downloaded the .zip file, extrated and started it, clicked on "load installed UWP apps", selected the two app I wanted to add (halo and forza), click on the export button, it didn't work and I received an error message about an api key but I dont have one, the app closed itself. I started it back again, redid the same thing, and now it worked and told me to restart steam. And them when I did, Steam could no longer connect itself to it's network, no matter what I do and restart (both steam and the PC), I can't acces the store or see my friends online. I already removed both games from my steam library, but that didn't do anything. I assume I'll have to reinstall steam? But I'm afread it won't work and I broke something important, but I don't understand how, wtf could UWPHook have done ? I though I was just adding game to steam :\  I hope someone can help me! 

TLDR : Used UWPHook for the first time, and now steam can no longer connect online.
